usaspending_api/awards/v2/views/awards.py

```diff
@@ -21,3 +24,33 @@ def get(self, request):
         }
 
         return Response(response)
+
+
+class AwardRetrieveViewSet(APIDocumentationView):
+    """
+    endpoint_doc: /awards.md
+    """
+
+    def _parse_and_validate_request(self, generated_unique_award_id) -> dict:
```

Comment:
I recommend adding a ToDo comment for a future modification to allow both `generated_unique_award_id` or `id` as the provided identifier. It will most likely help with the transition period when frontend is switching from v1 to v2.
 
I don't know an obvious path to do this cleanly with TinyShield.